.pre-commit-config.yaml

```diff
@@ -37,14 +34,29 @@
     sha: f464d46
     hooks:
     -   id: pep257
-        exclude: tests/dist|cherrypy/(_cp|__init__|__main__|daemon|_helper|(lib/(__init__|auth|auth_basic|auth_digest|caching|covercp|cpstats|cptools|encoding|gctools|httpauth|httputil|jsontools|lockfile|locking|profiler|reprconf|sessions)|process/(plugins|servers|win32)|scaffold/__init__).py|test|tutorial)
+        exclude: |
```

Comment:
This is tightly coupled with the hack in `tox.ini` removing `exclude:` from here in order to list all files not yet converted.